mne/decoding/receptive_field.py

```diff
@@ -56,10 +57,15 @@ class ReceptiveField(BaseEstimator):
     ``delays_``: array, shape (n_delays,), dtype int
         The delays used to fit the model, in indices. To return the delays
         in seconds, use ``self.delays_ / self.sfreq``
-    ``keep_samples_`` : slice
+    ``valid_samples_`` : slice
         The rows to keep during model fitting after removing rows with
-        missing values due to time delaying.
+        missing values due to time delaying. This can be used to get an
+        output equivalent to that of ``mode='valid'`` of
```

Comment:
Feels a bit awkward, how about "This can be used to get an output equivalent to using :func:`numpy.convolve` or :func:`numpy.correlate` with ``mode='valid'``."?